s3torchconnectorclient/python/tst/integration/test_mountpoint_s3_integration.py

```diff
@@ -404,6 +404,44 @@ def test_delete_object_invalid_bucket(
         )
 
 
+@pytest.mark.parametrize("force_path_style", [False, True])
+def test_copy_object(sample_directory: BucketPrefixFixture, force_path_style: bool):
+    client = MountpointS3Client(sample_directory.region, TEST_USER_AGENT_PREFIX, force_path_style=force_path_style)
+
+    src_key = f"{sample_directory.prefix}hello_world.txt"
+    dst_key = f"{sample_directory.prefix}hello_world_COPY.txt"
+
+    # set up
+    client.delete_object(sample_directory.bucket, dst_key)
+
+    client.copy_object(src_key, dst_key)
+
+    src_obj = client.get_object(sample_directory.bucket, src_key)
+    dst_obj = client.get_object(sample_directory.bucket, dst_key)
+
+    assert dst_obj.key == dst_key
+    assert str(dst_obj) == str(src_obj)
```

Comment:
Is this meant to compare the object contents? I don't think it would do that.

Reply:
It wouldn't 🙂 tried and only `b''.join(obj)` does the job; updating